redash/models.py

```diff
@@ -658,10 +660,9 @@ def all_queries(cls, groups, drafts=False):
             .order_by(cls.created_at.desc())
 
         if drafts:
-            q = q.where(Query.name == 'New Query')
+            q = q.where(Query.is_draft == True)
         else:
-            q = q.where(Query.name != 'New Query')
-
+            q = q.where(Query.is_draft == False)
```

Comment:
Yeah, I was thinking that new queries would be drafts, and that maybe we could have a migration step that goes through the existing queries and marks as draft any queries where Query.name == 'New Query', so upon upgrade the set of draft queries wouldn't change.

The one thing we haven't addressed yet is the ability for folks to find draft queries through the search interface, since as is I think they won't show up at all unless they're explicitly promoted from draft to public. Did you have ideas re: how you want that to look, @arikfr?